src/internal/infrastructure/persistence/CLAUDE.md

```
@@ -1,33 +1,35 @@
-| Besoin | Package |
-|--------|---------|
-| Stocker des données clé-valeur | `storage/boltdb/` |
-| Charger la configuration YAML | `config/yaml/` |
+| Need | Package |
+|------|---------|
+| Store key-value data | `storage/boltdb/` |
+| Load YAML configuration | `config/yaml/` |
 
 ## Structure
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)